R/zzz.R

```diff
@@ -0,0 +1,9 @@
+.onLoad <- function(libname,pkgname){
+  cache_duration <- as.numeric(Sys.getenv("piggyback_cache_duration", unset = "600"))
+  assign(x = "pb_info",
+         value = memoise::memoise(pb_info, ~ memoise::timeout(cache_duration)),
+         envir = parent.env(environment()))
+  assign(x = "pb_releases",
+         value = memoise::memoise(pb_releases, ~ memoise::timeout(cache_duration)),
+         envir = parent.env(environment()))
```

Comment:
Nice! guessing this syntax should avoid CRAN mistakenly complaining that `<<-` assigns to the global env

Reply:
Yep exactly!